packages/react-query/src/__tests__/useQueries.test-d.tsx

```diff
@@ -136,8 +136,9 @@ describe('UseQueries config object overload', () => {
       ],
     })
 
-    const data = queryResults[0].data
+    const firstResult = queryResults[0]
 
-    expectTypeOf(data).toEqualTypeOf<number | undefined>()
+    expectTypeOf(firstResult).toEqualTypeOf<UseQueryResult<number, Error>>()
```

Comment:
With the addition of this test code, we’ll have a correct type inference test for skipToken.